packages/framework/presence/src/latestValueManager.ts

```diff
@@ -17,7 +19,6 @@ import type {
 	JsonSerializable,
 } from "@fluid-experimental/presence/internal/core-interfaces";
 import type { ISubscribable } from "@fluid-experimental/presence/internal/events";
-import { createEmitter } from "@fluid-experimental/presence/internal/events";
```

Comment:
Tangent: I'm a bit confused here. Presence does all its imports with the full package name instead of relative paths? (cc @jason-ha).

Reply:
core-utils also does: https://github.com/microsoft/FluidFramework/blob/main/packages/common/core-utils/src/test/timer.spec.ts#L12. I think that's to avoid adding eslint suppression.

Reply:
It isn't lint suppression.
The best use for "self-references" is in testing. We should test against the public API more. For that we should use package reference where possible. Then we can also see where we are violating API to do certain testing.
Here it is used to manage the temp duplication of the tree events package and workaround `api-extractor` fussiness. Everything in presence that is using one these complex presence/internal/foo paths has a plan to get rid of them.